### PR TITLE
Graphql schema

### PR DESCRIPTION
**Pull request by MandeeRiz**
@boggsey

### CODE REVIEW

File: schema.graphql [hunk 1]
```diff
@@ -0,0 +1,17 @@
+"A Quote is a phrase. It is made up of id, quote and author"
```
[the remark this answers is not in the document]
Comment (MandeeRiz) [reply schema.graphql]: I will take it out. Thanks for reviewing!

File: gqlgen/graph/schema.graphqls [hunk 2]
```diff
@@ -0,0 +1,27 @@
+#https://gqlgen.com/getting-started/
+
+"A Quote is a phrase. It is made up of id, quote and author"
+type Quote {
+   "The id of a quote"
```
Comment (boggsey) [inline gqlgen/graph/schema.graphqls:5]: I think you need to change the way the comments are created. They use # instead of ""

Comment (MandeeRiz) [reply gqlgen/graph/schema.graphqls]: Done! Thanks for pointing that out.

File: gqlgen/graph/schema.graphqls [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  }
+}
+
+type Query {
+  Quote{
+    id: String!
+    quote: String!
+    author: String!
+  }
+}
```
Comment (librairica) [inline gqlgen/graph/schema.graphqls:27]: You should only have one Query block

**Review by librairica — APPROVED** [review]
I pulled down your branch, ran your server, and it looks great!

**Pull request merged**